src/Standards/Generic/Docs/Functions/OpeningFunctionBraceKernighanRitchieStandard.xml

```diff
@@ -1,22 +1,21 @@
-<documentation title="Opening Brace in Function Declarations">
+<documentation title="Opening Function Brace Kerninghan Ritchie">
     <standard>
     <![CDATA[
-    Function declarations follow the "Kernighan/Ritchie style". The function brace is on the same line as the function declaration. One space is required between the closing parenthesis and the brace.
+    The function opening brace must be on the same line as the function declaration closing
+    parenthesis, with exactly one space between them, and must be the last content on the line.
     ]]>
     </standard>
     <code_comparison>
-        <code title="Valid: Brace on same line.">
+        <code title="Valid: Opening brace on the same line.">
         <![CDATA[
 function fooFunction($arg1, $arg2 = '')<em> {</em>
-    ...
```

Comment:
I would instead replace this with something along the lines of either `doSomething()` or `// Do something.`. That avoids getting into a discussion about whether the close brace of a function should directly follow the open brace for empty function or should be on a new line. I know that's not what this sniff is about, but let's keep the example clear-cut and not cause extra cognitive overhead.

(here and below)

Reply:
That is a good point. Thanks for your suggestion. I just pushed a new commit adding `// Do something.` to the function body in both code examples.